src/core-io/include/mp-units/ostream.h

```diff
@@ -34,7 +34,7 @@ namespace detail {
 template<typename CharT, class Traits, auto R, typename Rep>
 void to_stream(std::basic_ostream<CharT, Traits>& os, const quantity<R, Rep>& q)
 {
-  os << q.number();
+  os << std::to_string(q.number());
```

Comment:
```suggestion
  using std::to_string;
  os << to_string(q.number());
```

Reply:
I hesitate to use "using" in public headers, as this pulls in namespaces also for users of the library.